bin/des-pizza-cutter-prep-tile

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+import os
+import sys
+import logging
+import desmeds
+
+import jinja2
+import yaml
+from pizza_cutter.des_pizza_cutter import add_extra_des_coadd_tile_info
+from pizza_cutter.files import makedir_fromfile, get_des_info_file
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    '--config',
+    required=True,
+    help='configuration file',
+)
+parser.add_argument(
+    '--tilename',
+    required=True,
+    help='tile to process',
+)
+parser.add_argument(
+    '--band',
+    help='band to process',
+    required=True,
+)
+parser.add_argument(
+    '--log-level',
+    default='warning',
+    help=(
+        'python logging level [one of critical error, '
+        'warning, info, or debug]'),
+)
+parser.add_argument(
+    '--clean',
+    action='store_true',
+    help=('remove all the input source files'),
+)
+
+
+def main():
+    """Generate the information needed for pizza slicing a DES Y3+ coadd tile.
+    """
+
+    args = parser.parse_args()
+
+    logging.basicConfig(stream=sys.stdout)
+    logging.getLogger('pizza_cutter').setLevel(
+        getattr(logging, args.log_level.upper()))
+
+    print('tilename:', args.tilename, flush=True)
+    print('band:', args.band, flush=True)
+    with open(args.config, 'r') as fp:
+        _config = fp.read()
+
+    config_text = jinja2.Template(_config).render(
+        tilename=args.tilename,
+        band=args.band,
+    )
+    cfg = yaml.load(config_text, Loader=yaml.Loader)
+
+    cfg['medsconf'] = os.path.basename(args.config).replace('.yaml', '')
+
+    prep = desmeds.desdm_maker.Preparator(
+        cfg,
```

Comment:
Which entries in the config file is the `Preparator` using?

Reply:
I added a comment